### PR TITLE
Update Rule “do-you-display-information-consistently/rule”

### PR DESCRIPTION
**Pull request by uly1**
Automatically generated by Decap CMS

### CODE REVIEW

File: rules/do-you-display-information-consistently/rule.md [hunk 1]
```diff
@@ -1,27 +1,25 @@
...
 
+![Figure: Good Example - This text is taken from where the "More on SSW Chinafy" link goes](homepage-with-link-to-chinafy.png)
 
-::: good  
-![Figure: Good example - Click "More" and the next page begins with the same information](../../assets/ConsistentInformation.gif)  
-:::
+![Figure: Good Example - You can see the text again here, at the link's destination](chinafy-page.png)
 
-As an example, the SSW website follows a consistent order: a summary of the product/service and its benefit followed by a longer paragraph with more information. When we want to redirect users to a page we simply use the summary part of that page as narrative on the other page (e.g. events list). As a result, user will have a nice flow between pages when navigating around the site.     [See a live example](http://www.ssw.com.au/ssw/Events/).
+As an example, the [SSW website](https://ssw.com.au/) follows a consistent order: a summary of the product/service and its benefit, followed by more information o that topic.
```
Comment (bettybondoc) [inline rules/do-you-display-information-consistently/rule.md:21]: @uly1 

- [ ] Please fix typo

Change from
   followed by more information o that topic.
to
    followed by more information **_on_** that topic.

File: rules/do-you-display-information-consistently/rule.md [hunk 2]
```diff
@@ -1,27 +1,25 @@
 ---
 type: rule
-archivedreason: 
 title: Do you display information consistently?
-guid: 5b878864-7ced-4b4c-a979-adadf38c7363
 uri: do-you-display-information-consistently
-created: 2015-02-16T01:50:14.0000000Z
 authors: []
```
Comment (bettybondoc) [inline rules/do-you-display-information-consistently/rule.md:5]: @uly1

- [ ] please add author

File: rules/do-you-display-information-consistently/rule.md [hunk 3]
```diff
@@ -1,27 +1,25 @@
...
+As an example, the [SSW website](https://ssw.com.au/) follows a consistent order: a summary of the product/service and its benefit, followed by more information o that topic. 
 
-Please Note: the figure here is intended to show the consistency in the text, the use of a link called More is not desirable as per our Google Rule     [Do your links have relevancy to the page you're linking to?](http://www.ssw.com.au/ssw/Standards/Rules/RulesToBetterGoogleRankings.aspx#Relevancy)
+When we want to redirect users to a page, we simply use the summary part of that page as narrative on the other page. As a result, the user will have a nice flow between pages when navigating around the site.
 
-Please Note #2: This is about the text being consistent. However you should also keep your links consistent.
+Tip: In this scenario where the summary serves as a teaser for more detialed content, have the link at the end of the section, so it's the natural next step for someone who has read it and wants to know more.
```
Comment (bettybondoc) [inline rules/do-you-display-information-consistently/rule.md:25]: @uly1 

- [ ] please fix typo

From
detialed
To
detailed

**Review by bettybondoc — CHANGES_REQUESTED** [review]

File: rules/do-you-display-information-consistently/rule.md [hunk 4]
```diff
@@ -1,27 +1,25 @@
...
+archivedreason: null
+guid: 5b878864-7ced-4b4c-a979-adadf38c7363
 ---
-
-When redirecting users to another page make sure that the       text is consistent between the two pages.
+When redirecting users to another page, make sure that the text is consistent between the two pages.
 
 <!--endintro-->
 
+![Figure: Good Example - This text is taken from where the "More on SSW Chinafy" link goes](homepage-with-link-to-chinafy.png)
```
Comment (bettybondoc) [inline rules/do-you-display-information-consistently/rule.md:17]: @uly1 

- [ ] Do we need the "::: good   ...   :::" to get the green tick?

Comment (uly1) [reply rules/do-you-display-information-consistently/rule.md:17]: yes i think so

**Review by bettybondoc — APPROVED** [review]

**Pull request merged**